nucleus-core/src/main/java/io/github/nucleuspowered/nucleus/modules/afk/listeners/AFKRotationOnlyListener.java

```diff
@@ -39,7 +39,7 @@ public void onPlayerMove(final RotateEntityEvent event, @Root Player player,
     public boolean shouldEnable(INucleusServiceCollection serviceCollection) {
         AFKConfig.Triggers triggers = serviceCollection.moduleDataProvider().getModuleConfig(AFKConfig.class)
                 .getTriggers();
-        return triggers.isOnMovement() && !triggers.isOnRotation();
```

Comment:
This commit was part of another PR and should be removed from here.

Reply:
Was tired and didn't realise this commit slipped in